group-aws/bestpractices.md

```diff
@@ -0,0 +1,19 @@
+# Best practices
```

Comment:
We welcome the improvements from everyone! But can document that more explicitly too.